toqito/states/max_mixed.py

```diff
@@ -43,14 +43,15 @@ def max_mixed(dim: int, is_sparse: bool = False) -> [np.ndarray, sparse.dia_matr
 
     >>> from toqito.states import max_mixed
     >>> max_mixed(2, is_sparse=False)
-    [[0.5, 0. ],
-     [0. , 0.5]]
+    array([[0.5, 0. ],
+           [0. , 0.5]])
+
 
     One may also generate a maximally mixed state returned as a sparse matrix
 
     >>> from toqito.states import max_mixed
-    >>> max_mixed(2, is_sparse=True)
-        <2x2 sparse matrix of type '<class 'numpy.float64'>'
+    >>> max_mixed(2, is_sparse=True) # doctest: +ELLIPSIS, +NORMALIZE_WHITESPACE
+    <2x2 sparse matrix of type '<class 'numpy.float64'>'
```

Comment:
Hmm. I see. Wonder if this will not be an issue when/if we switch from `matrix` to `np.array` objects universally.

Reply:
> Wonder if this will not be an issue when/if we switch from matrix to np.array objects universally.

Don't know.

I thought doctest was being very particular about the whitespace. Maybe 1 is supposed to be a tab while the other is not. `ruff` did not like it when I used tab to create the whitespace.